compiler/src/symboltable.rs

```diff
@@ -726,8 +726,15 @@ impl SymbolTableBuilder {
     fn scan_string_group(&mut self, group: &ast::StringGroup) -> SymbolTableResult {
         match group {
             ast::StringGroup::Constant { .. } => {}
-            ast::StringGroup::FormattedValue { value, .. } => {
+            ast::StringGroup::FormattedValue {
+                value,
+                conversion: _,
```

Comment:
I think this is what's causing the failure, in both this and the other PR. You can just add `..` after the fields you actually use.

Reply:
I fixed that at #1625. You can ignore the changes there as I will rebase once it is in.

Reply:
Ah okay, I didn't realize that that was passing now; I merged it.